src/main/java/frc/robot/subsystems/ClimbSubsystem.java

```diff
@@ -88,15 +87,13 @@ public void stopMotors() {
     backMotor.stopMotor();
   }
 
-  public boolean isAtClimbVoltage() {
-    return climbFrontVoltageSignal.refresh()
-        .getValue()
-        .minus(climbControlFront.getOutputMeasure())
-        .abs(Volts) <= CLIMB_VOLTAGE_TOLERANCE.in(Volts)
-        && climbBackVoltageSignal.refresh()
-            .getValue()
-            .minus(climbControlBack.getOutputMeasure())
-            .abs(Volts) <= CLIMB_VOLTAGE_TOLERANCE.in(Volts);
+  /**
+   * Method to determine if the climb motors are moving
+   * 
+   * @return true if the motor has moved on rotation
+   */
+  public boolean areClimbMotorsMoving() {
+    return (climbFrontPositionSignal.refresh().getValue().in(Rotations) > 1);
```

Comment:
Maybe you want to check the velocity. It's possible the motor starts at a position > 1 and it doesn't move, and your test would pass.
Also, the comment and method name suggest this tests if both motors are moving, but it only checks one motor. The climb motors aren't mechanically attached, they run separately.